relay-quotas/Cargo.toml

```diff
@@ -23,7 +23,7 @@ failure = { version = "0.1.8", optional = true }
 log = { version = "0.4.8", optional = true }
 relay-common = { path = "../relay-common" }
 relay-redis = { path = "../relay-redis", optional = true }
-sentry = { version = "0.20.1", optional = true }
+sentry = { git = "https://github.com/getsentry/sentry-rust", optional = true }
```

Comment:
Can we bump this to the next released version instead of a git dependency?

Reply:
The goal basically was to dogfood the new log integration before we publish it properly. wdyt?